app/models/person_upload.rb

```diff
@@ -0,0 +1,32 @@
+require 'virtus'
+class PersonUpload
+  include Virtus.model
+  extend ActiveModel::Naming
+  include ActiveModel::Conversion
+  include ActiveModel::Validations
+
+  attr_accessor :file, :group_id
+  attr_reader :import_count, :csv_errors
+
+  validates :file, presence: true
+  validates :group_id, presence: true
+
+  def initialize(*)
+    @csv_errors = []
+    super
+  end
+
+  def save
+    return nil unless valid?
+
+    groups = Group.where(id: group_id)
+    importer = CsvPeopleImporter.new(file.read, groups: groups)
+    @import_count = importer.import
+    @csv_errors = importer.errors
+    @import_count
```

Comment:
I tried to avoid touching `CsvPeopleImporter` too much. Perhaps it would make more sense like this:

```ruby
@import_count = importer.import || 0
# ...
@import_count > 0
````